webrender/src/frame.rs

```diff
@@ -1131,11 +1133,13 @@ pub fn build_scene(config: &FrameBuilderConfig, request: SceneRequest) -> BuiltS
         &mut pipeline_epoch_map
```

Comment:
couldn't we just pass `&mut request.scene.pipeline_epochs` here?

Reply:
The scene flattening reads from a pipeline epoch map, and builds a new one (which has the effect of filtering out pipeline epochs that are not in the scene), so there has to be two epoch maps at some point. That said I agree that the way it is presented here isn't very obvious, I'll look into making this a bit nicer.